lib/mayaHydra/hydraExtensions/CMakeLists.txt

```
@@ -30,31 +30,54 @@ set(HEADERS
 # -----------------------------------------------------------------------------
 # compiler configuration
 # -----------------------------------------------------------------------------
-target_compile_definitions(${TARGET_NAME} 
-    PRIVATE 
-        MAYAHYDRALIB_EXPORT
-        $<$<BOOL:${IS_MACOSX}>:OSMac_>
-        # Copy-pasted from lib/mayaUsd/CMakeLists.txt
-)
-
+if(DEFINED MAYAUSD_LOCATION)
+    target_compile_definitions(${TARGET_NAME} 
+        PRIVATE 
+            MAYAHYDRALIB_EXPORT
+            MAYAHYDRALIB_MAYAUSD_IS_USED_TO_BUILD
```

A bit wordy for a boolean variable.  How about:
```suggestion
            MAYAHYDRALIB_MAYAUSD_ENABLED
```